crates/physical-plan/src/plan.rs

```diff
@@ -1561,11 +1653,20 @@ mod tests {
             None,
         ));
 
-        let db = SchemaViewer {
-            schemas: vec![t.clone()],
-        };
+        let db = SchemaViewer::new(vec![t.clone()]).optimize(true);
 
         let sql = "select * from t where x = 3 and y = 4 and z = 5";
+        check_sub(
+            &db,
+            sql,
+            expect![
+                r#"
+Index Scan using Index id 2: (x, y, z) on t
```

Comment:
```suggestion
Index Scan using index id: 2
```

Can we must mention the index id in this part, and then have the index info in the metadata section?